airflow/providers/odbc/hooks/odbc.py

```diff
@@ -24,7 +24,23 @@
 import pyodbc
 
 from airflow.hooks.dbapi_hook import DbApiHook
-from airflow.utils.helpers import merge_dicts
+
+
+# We do not import it from airflow.utils because merge_dicts is not available in Airflow 1.10
+# So this operator would not be backportable
+def merge_dicts(dict1, dict2):
```

Comment:
I understand this change comes from master to make things compatible, but think we should remove it entirely (in another PR). This is a one-liner in native Python: `{**dict1, **dict2}`, no need for a custom dict merge function.

Reply:
``{**dict1, **dict2}`` only merges the first dictionary. This function also merges nested dictionaries. 
```
>>> dict_a = {'A': {'B': {'C': 'D'}}}
>>> dict_b = {'A': {'B': {'E': 'F'}}}
>>> {**dict_a, **dict_b}
{'A': {'B': {'E': 'F'}}}
>>> merge_dicts(dict_a, dict_b)
{'A': {'B': {'C': 'D', 'E': 'F'}}}
```

Reply:
Ah I see, read it too quick.